1.现在api是星号隐藏的，处理成默认呈现的
2.现在要新加入一个大模型聊天的面板，我联通测试的时候，大模型返回的语言可以在聊天面板看到
3.我点击生成ai报告，有几次会成功调用大模型，但是会显示处理失败：IdeaDiscovery.tsx:81 处理失败: 
{message: '网络连接失败，请检查网络设置', type: 'NETWORK_ERROR', code: 'NETWORK_ERROR', timestamp: Sat Jul 26 2025 08:06:02 GMT+0800 (中国标准时间)}
code
: 
"NETWORK_ERROR"
message
: 
"网络连接失败，请检查网络设置"
timestamp
: 
Sat Jul 26 2025 08:06:02 GMT+0800 (中国标准时间)
[[Prototype]]
: 
Object
type
: 
"NETWORK_ERROR"
[[Prototype]]
: 
Object
constructor
: 
ƒ Object()
hasOwnProperty
: 
ƒ hasOwnProperty()
isPrototypeOf
: 
ƒ isPrototypeOf()
propertyIsEnumerable
: 
ƒ propertyIsEnumerable()
toLocaleString
: 
ƒ toLocaleString()
toString
: 
ƒ toString()
valueOf
: 
ƒ valueOf()
__defineGetter__
: 
ƒ __defineGetter__()
__defineSetter__
: 
ƒ __defineSetter__()
__lookupGetter__
: 
ƒ __lookupGetter__()
__lookupSetter__
: 
ƒ __lookupSetter__()
__proto__
: 
(...)
get __proto__
: 
ƒ __proto__()
set __proto__
: 
ƒ __proto__()
p	@	IdeaDiscovery.tsx:81
